lua/mini/jump.lua

```diff
@@ -388,6 +392,11 @@ function H.highlight(pattern)
     return
   end
 
+  -- Do not highlight in operator-pending mode, as it does not support repeating.
+  if string.find(MiniJump.state.mode, 'o') then
```

Comment:
Let's not do that. Although `fFtT` is not repeatable in this mode, there is a dot-repeat. It proved to be useful when doing something like `dte` and then `.` to repeat deleting.

Reply:
Ahh, dot repeating is a good point! I discarded this commit.